tests/scripts/all.sh

```diff
@@ -1608,6 +1608,19 @@ component_test_memsan_constant_flow () {
     make test
 }
 
+skip_non_constant_flow_components () {
+    # The following components are especially slow and don't have any constant-flow annotations,
+    # so disable them for better CI performance.
+    # It would be better to pass a list of tests that we want to run, rather than tests to skip,
+    # but this functionality is currently missing from run-test-suites.pl.
```

Comment:
`run-test-suites.pl` is not involved here: this shell function is only called during builds that use cmake.